backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/starrating/BpkStarRating.kt

```diff
@@ -93,24 +113,51 @@ private fun BpkStarRating(
   rating: Float,
   maxRating: Int,
```

Comment:
Let's have 2 arguments: numOfStars (for num of stars) and and maxRating (for content description)